### ISSUE TITLE
DC line model breaks PWL generator cost format

### ISSUE THREAD

**Issue by claytonpbarrows** [lanl-ansi/PowerModels.jl#316]
e.g. [case3_pwlc.zip](https://github.com/lanl-ansi/PowerModels.jl/files/2135479/case3_pwlc.zip)
the file with the dc line field results in the following error:
```
network_data = PowerModels.parse_file("case3_pwlc_dc.m")
[warn | InfrastructureModels]: Matlab parser skipping the following line:
  ]
[warn | InfrastructureModels]: Matlab parser skipping the following line:
  ]
[warn | PowerModels]: added zero cost function data for dclines
[warn | PowerModels]: added zero cost function data for dclines
[warn | PowerModels]: pwl x value 0.12 is outside the generator bounds 0.0-0.0
[warn | PowerModels]: pwl x value 0.12 is outside the generator bounds 0.0-0.0
[warn | PowerModels]: pwl x value 0.16 is outside the generator bounds 0.0-0.0
[warn | PowerModels]: pwl x value 0.16 is outside the generator bounds 0.0-0.0
[warn | PowerModels]: pwl x value 0.2 is outside the generator bounds 0.0-0.0
[warn | PowerModels]: pwl x value 0.2 is outside the generator bounds 0.0-0.0

non-increasing x values in pwl cost model
```

**Comment by ccoffrin** [comment]
Confirmed this is a bug, thanks for finding it!  Should be fixed in master now.  If you would like to apply the fix locally, only a few lines need to be updated, have a look at https://github.com/lanl-ansi/PowerModels.jl/pull/317.

**Comment by jd-lara** [comment]
@claytonpbarrows is it possible to update the branch `moi-julia-v0.7` to include this fix?

**Comment by ccoffrin** [comment]
Short answer, not for some time.

When Ipopt is woking again with MOI, I will try to update the `moi` branch to PowerModels master.  Then you can look at merging that in `moi-julia-v0.7`.  If you need a quick fix, you can re-do #317 for on `moi-julia-v0.7`.
